MekHQ/src/mekhq/campaign/MercRosterAccess.java

```diff
@@ -473,10 +473,10 @@ private void writePersonnelData() {
                 preparedStatement.setInt(6, forceId);
                 preparedStatement.setInt(7, 1);
                 //TODO: get joining date right
-                preparedStatement.setDate(8, new java.sql.Date(p.getBirthday().getTimeInMillis()));
+                preparedStatement.setDate(8, java.sql.Date.valueOf(p.getBirthday()));
```

Comment:
I hate to ask, but is this going to impact this type of export? Not that I've ever even heard of a MercRoster database, but...

Reply:
MercRoster that is a name I haven't heard in a long time. It was one of Taharqa projects.

https://sourceforge.net/projects/mercroster/

Reply:
This method was added for reverse compatibility, so it shouldn't?
https://www.javatpoint.com/java-sql-date-valueof-method

Reply:
As for MercRoster, see Hammer's reply... although I've seen it surprisingly often when looking at personnel (probably because of how tied in the two are)